app/scripts/controllers/transactions/index.js

```diff
@@ -1474,7 +1481,7 @@ export default class TransactionController extends EventEmitter {
     }
   }
 
-  _buildEventFragmentProperties(txMeta, extraParams) {
+  async _buildEventFragmentProperties(txMeta, extraParams) {
```

Comment:
[Q] Were all usages of this function converted to use await?

Reply:
yep